quiz.py

```diff
@@ -1,23 +1,19 @@
 def start_quiz(questions):
     score = 0
-    for i in range(0, len(questions)):
-        print(questions[i]["question"]+"\n")
+    for index, question in enumerate(questions):
```

Comment:
This enumerate is redundant as you have not used that index variable anywhere in the code. Though iterating over questions instead of index is a good choice. Kudos to that change.